src/renderer/components/SaveButton/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export {default} from './SaveButton';
```

Comment:
Doesn't seem like this is necessary, we should be able to just directly import this.

Reply:
Done